views/index.html

```diff
@@ -41,7 +41,7 @@ <h3>Example Output:</h3>
       </div>
       <div class="footer">
         <p>
-          by <a href="http://www.freecodecamp.com/">freeCodeCamp</a> | <a href="https://hyperdev.com/#!/remix/FCC_API_timestamp_bplate/8338be2-a353-4892-8a0d-6295e4f12eb5">Click here to Remix</a>
+          by <a href="http://www.freecodecamp.com/">freeCodeCamp</a>
```

Comment:
```suggestion
          by <a href="http://www.freecodecamp.org/">freeCodeCamp</a>
```
Might as well go directly.

Reply:
Shoot, thank you for catching that. Definitely something that needs to be fixed on the working example project, too.